lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -113,6 +120,51 @@
   </div>
 %endif
 
+%if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS') and section_data['access']['instructor']:
+    <hr>
+    <div class="course-specific-container action-type-container">
+        <h2>${_('Submit External Grades')}</h2>
+        <div class="request-response-error"></div>
```

Comment:
I'm not quite sure what this does, so I can't give a recommendation right now. Can I get a demo @azimpradhan ?